test/TestServerProjects/body-boolean/Generated/Models/Error.Serialization.cs

```diff
@@ -20,6 +21,7 @@ internal static void Serialize(Error model, Utf8JsonWriter writer)
                 writer.WritePropertyName("message");
                 writer.WriteStringValue(model.Message);
             }
+
```

Comment:
It seems there is an extra carriage return added to both the Serialize and Deserialize methods.